arbitrary-users-patch/base_images

```diff
@@ -1,18 +1,17 @@
-che-antora-2.3          docker.io/antora/antora:2.3.3
-che-cpp-rhel7           registry.access.redhat.com/devtools/llvm-toolset-rhel7
-che-dotnet-2.2          mcr.microsoft.com/dotnet/core/sdk:2.2-stretch
-che-dotnet-3.1          mcr.microsoft.com/dotnet/core/sdk:3.1.301-buster
-che-golang-1.12         golang:1.12-stretch
-che-golang-1.14         golang:1.14-stretch
-che-java11-gradle       gradle:6.6.1-jdk11
-che-java11-maven        maven:3.6.3-jdk-11
-che-java8-maven         maven:3.6.1-jdk-8
-che-nodejs10-community  node:10.16
-che-nodejs12-community  node:12.18
-che-nodejs10-ubi        registry.access.redhat.com/ubi8/nodejs-10
-che-nodejs8-centos      registry.centos.org/che-stacks/centos-nodejs
-che-php-7               quay.io/eclipse/che-php-base:7.4
-che-python-3.6          centos/python-36-centos7:1
-che-python-3.8          python:3.8.6-slim
-che-quarkus             quay.io/quarkus/centos-quarkus-maven:21.0-java11
-che-rust-1.39           rust:1.39.0-slim
+che-antora-2.3          docker.io/antora/antora@sha256:fbf31b3be4e5dd215c82f9b5b403c5e746f68bcf50ced9b7d41b6d1414aa5714
+che-cpp-rhel7           registry.access.redhat.com/devtools/llvm-toolset-rhel7@sha256:24573aefd172be551ab4ae047a31f5ba5cd59f2e5bc207188f3b7045ac9d7c72
+che-dotnet-2.2          mcr.microsoft.com/dotnet/core/sdk@sha256:db9b38d066fdbdd5ac5ce862076c27e4ae17f2b57cbacd03bdc3b24e06696ae8
+che-dotnet-3.1          mcr.microsoft.com/dotnet/core/sdk@sha256:c87fd21bdebb2c2d573ecd703981476e5b8ac6a0e96d134722bb672e1c231fea
+che-golang-1.12         golang@sha256:8b0f4fee390ca1308ef5c522accad9f7d98a4134fcde1b2cff33df1cbb1b3b97
+che-golang-1.14         golang@sha256:a60bc0c6be7e352336044e68db7089d8604864933fcbc0d7cb2285d60b0b2d98
+che-java11-gradle       gradle@sha256:c25f15608cd9bee8e290dd5654f9a784bea4505719fdedd0ad7ac640bed0fc50
+che-java11-maven        maven@sha256:1d29ccf46ef2a5e64f7de3d79a63f9bcffb4dc56be0ae3daed5ca5542b38aa2d
+che-java8-maven         maven@sha256:69b40237b342fee9bf996b81110f4dab250a4bc6a2ee52866965101eda066324
+che-nodejs10-community  node@sha256:f4b6f471cdd4b66b27eef899e7f8423ecd9fbfc863b2cb7a59978a7f64c8e0c3
```

Comment:
personal pet peeve: why don't we mention the registry here? should be docker.io/... not just image name. That way you'll never accidentally get the wrong image if it happens to be on multiple registries and your docker config file picks the wrong registry first.

Reply:
Good point :+1: